_docs/installation/linux.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Instalacion Windows
+description: Getting started with Docsy Jekyll

```

Comment:
Change description

Reply:
ready with rar file and commands. 
file's name = EVAB_linux_ubuntu2004lts.rar
folder's name = EVAB_ACM_UCSP

Maybe We need follow the standard

Reply:
Windows and macos zip file is only for test. Linux is ok and ready to download.